types/index.d.ts

```diff
@@ -192,7 +192,9 @@ declare var api: {
     };
     tickPeriod: {
         get(name: string, settings?: { utc?: boolean }): { cast: (d: Date) => Date; next: (d: Date) => Date; };
-        add(name: string, period: { cast: (d: Date) => Date; next: (d: Date) => Date; }, settings?: { utc?: boolean }): void;
```

Comment:
This line was too long according to the `tslint.json` file (`120` characters max)